### ISSUE TITLE
[Q] Cohorts.cohortId vs Cohorts.id

### ISSUE THREAD

**Issue by redmitry** [ga4gh-beacon/beacon-v2-Models#102]
Hello,

All beacons except "cohorts" use "id" property for their identifiers: [cohorts/defaultSchema.json](https://github.com/ga4gh-beacon/beacon-v2-Models/blob/c41c75728b3fbff90fd57a9bfff58c74218e5932/BEACON-V2-Model/cohorts/defaultSchema.json#L22)  
~~~json
"properties": {
    "cohortId": {
      "type": "string",
      "examples": ["cohort-T2D-2010"]
    },
~~~
is there any particular reason for this?

Cheers,

Dmitry

**Comment by mrueda** [comment]
...and what about all `cohort` terms using the prefix `cohort`. Should we remove the prefix for consistency?

Cheers,

Manu

**Comment by redmitry** [comment]
Do you mean cohorts data stored? If the spec. changes, all data should be updated consequently...

**Comment by mrueda** [comment]
Yep, spec. I already raised the topic internally a week ago. The nomenclature for cohort terms seems legacy...

**Comment by jrambla** [comment]
Addressed in #103
